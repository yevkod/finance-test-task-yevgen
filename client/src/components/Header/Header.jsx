import React from 'react';
import s from './Header.module.css';

export const Header = () => {
    return (
        <div>
            <div className={s.logo}>
                <span>Incode</span>
                <span className={s.finance}>Finance</span>
            </div>
        </div>
    );
};


