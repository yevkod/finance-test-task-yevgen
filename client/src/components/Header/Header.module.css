.logo {
    display: flex;
    align-items: flex-start;
}

.finance {
    color: #808080;
}

.logo > span {
    font-size: 52px;
    margin-top: -7px;
    margin-left: 25px;
}

@media screen and (max-width: 600px) {
    .logo > span {
        display: none;
    }
}

@media screen and (max-width: 480px) {
    .logo > img {
        width: 150px;
    }
}
