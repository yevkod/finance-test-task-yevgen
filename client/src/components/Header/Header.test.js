import { render } from "@testing-library/react";
import {Header} from "./Header";

describe("<Header /> component", () => {
    it("should match snapshot", () => {
        const { container } = render(<Header />);
        expect(container).toMatchSnapshot();
    });
});
