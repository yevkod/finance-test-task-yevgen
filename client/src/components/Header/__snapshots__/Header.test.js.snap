// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component should match snapshot 1`] = `
<div>
  <div>
    <div
      class="logo"
    >
      <span>
        Incode
      </span>
      <span
        class="finance"
      >
        Finance
      </span>
    </div>
  </div>
</div>
`;
